config-controller/pkg/client/client.go

```diff
@@ -167,6 +167,7 @@ func (gc *grpcClient) PostPolicy(ctx context.Context, policy *storage.Policy) (*
 	req := &v1.PostPolicyRequest{
 		Policy:                 policy,
 		EnableStrictValidation: true,
+		PolicyAsCode:           true,
```

Comment:
This is not needed.  The policy source fields in the policy proto will give you the information if this policy is maintained as code or not.